pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -31,6 +31,7 @@ var bundleCheck = Check{
 	labels: None,
 }
 
+// nolint
 var genericPreflightChecks = []Check{
```

Comment:
This can be moved to `preflight_checks_nonwin.go`

Reply:
Done